docs/Quickstart.md

```diff
@@ -43,28 +43,31 @@ git remote add heroku git@heroku.com:appinfive.git
 
 4. Add ShopifyApp to gemfile
 ----------------------------
-```
+```sh
 $ echo "gem 'shopify_app'" >> Gemfile
-
-bundle install
+$ bundle install
 ```
 
 Note - its recommended to use the latest released version. Check the git tags to see the latest release and then add it to your Gemfile e.g `gem 'shopify_app', '~> 7.0.0'`
 
 5. Run the ShopifyApp generator
 -------------------------------
+```sh
+# use the keys from your app in the partners area
+$ rails generate shopify_app --api_key a366cbafaccebd2f615aebdfc932fa1c --secret 8750306a895b3dbc7f4136c2ae2ea293
+$ git add .
+$ git commit -m 'generated shopify app'
 ```
-use the keys from your app in the partners area
-rails generate shopify_app --api_key a366cbafaccebd2f615aebdfc932fa1c --secret 8750306a895b3dbc7f4136c2ae2ea293
-git add .
-git commit -m 'generated shopify app'
-```
+
+If you forget to set your keys or redirect uri above you will find them in the shopify_app initializer at: `/config/initializers/.`
```

Comment:
I'd write the full initializer name, as it's hardcoded and won't be changing anytime soon (or ever): `config/initializers/shopify_app.rb` (also put the dot '.' out of backticks)

Reply:
Good catch! Will be in next commit pending any further feedback.